submissions/antonkottans/frogger/app.js

```diff
@@ -0,0 +1,80 @@
+const Enemy = function (x, y, speed) {
+    this.x = x;
+    this.y = y;
+    this.speed = speed;
+    this.sprite = "images/enemy-bug.png";
+};
+const field = {
+    x: 4,
+    y: 4,
+};
+
+Enemy.prototype.update = function (dt) {
+    if (this.x >= (field.x+1) * 100) this.x = -100;
+    else this.x += dt * this.speed;
+
+    this.checkCollision();
+};
+
+Enemy.prototype.checkCollision = function () {
+    if (
+        Math.abs(player.x * 100 - this.x) <= 70 &&
+        Math.abs(player.y * 80 + 50 - this.y) < 20
+    ) {
+        player.x = player.start.x;
+        player.y = player.start.y;
+    }
+};
+
+Enemy.prototype.render = function () {
+    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+};
+
+const player = {
```

Comment:
Why do you write Enemy entity in OOP style and player not in OOP style? Try to keep the same code style throughout the project.